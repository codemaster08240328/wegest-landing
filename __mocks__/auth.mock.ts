/**
 * Email: testusergmail.com
 * password: testuser123
 */

export const signin = async ({
  email,
  password,
}: {
  email: string
  password: string
}) => {
  return new Promise((resolve) => {
    setTimeout(() => {
      resolve(1)
    }, 200)
  }).then(() => {
    if (email === 'testuser@gmail.com' && password === 'testuser123') {
      return {
        token: 'auth.token',
        user: {
          email: 'testuser@gmail.com',
          firstName: 'James',
          lastName: 'Wo',
          gender: 'Male',
          phone: 'xxxxx',
          role: 'user',
        },
      }
    }

    return 'invalid'
  })
}
