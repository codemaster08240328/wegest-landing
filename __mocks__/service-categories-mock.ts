export const categories = [
  {
    title: 'Colore2',
    backgroundColor: '1dc499',
    textColor: 'FFFFFF',
    services: [
      { id: '143BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '1638DB9B', title: 'Colore completo', qualified: true },
      { id: '143BDC9051', title: 'Colore Future', qualified: true },
      { id: '1630D59354', title: 'colore moda estate', qualified: false },
      { id: '163ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '123FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '1D3FD490', title: 'Deco', qualified: false },
      { id: '123AD49254', title: 'elumen', qualified: false },
      { id: '1C38DF9658', title: 'Giallo canarino', qualified: false },
      { id: '1430DE9A59', title: 'Manicure 5', qualified: false },
      { id: '1D38DE9559', title: 'Prova', qualified: false },
      { id: '1738DA9559', title: 'prova', qualified: false },
      { id: '143BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Body',
    backgroundColor: '2080e9',
    textColor: 'FFFFFF',
    services: [
      { id: '243BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '2638DB9B', title: 'Colore completo', qualified: true },
      { id: '2D38DE9559', title: 'Prova', qualified: false },
      { id: '2738DA9559', title: 'prova', qualified: false },
      { id: '243BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Aesthetics',
    backgroundColor: '1a7ea0',
    textColor: 'FFFFFF',
    services: [
      { id: '343BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '3638DB9B', title: 'Colore completo', qualified: true },
      { id: '343BDC9051', title: 'Colore Future', qualified: true },
    ],
  },
  {
    title: 'Meches',
    backgroundColor: 'ffb300',
    textColor: 'FFFFFF',
    services: [
      { id: '443BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '4638DB9B', title: 'Colore completo', qualified: true },
      { id: '443BDC9051', title: 'Colore Future', qualified: true },
      { id: '4630D59354', title: 'colore moda estate', qualified: false },
      { id: '463ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '423FDA9A', title: 'Colore senza ammonia', qualified: false },
    ],
  },
  {
    title: 'Fold',
    backgroundColor: '607d8b',
    textColor: 'FFFFFF',
    services: [
      { id: '543BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '543BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Solarium',
    backgroundColor: 'ffe903',
    textColor: 'FFFFFF',
    services: [
      { id: '643BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '643BDC9051', title: 'Colore Future', qualified: true },
      { id: '6630D59354', title: 'colore moda estate', qualified: false },
      { id: '623AD49254', title: 'elumen', qualified: false },
      { id: '6C38DF9658', title: 'Giallo canarino', qualified: false },
      { id: '6430DE9A59', title: 'Manicure 5', qualified: false },
      { id: '6738DA9559', title: 'prova', qualified: false },
      { id: '643BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Cut',
    backgroundColor: '64b5f6',
    textColor: 'FFFFFF',
    services: [
      { id: '743BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '7638DB9B', title: 'Colore completo', qualified: true },
      { id: '763ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '723FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '7D3FD490', title: 'Deco', qualified: false },
      { id: '7738DA9559', title: 'prova', qualified: false },
      { id: '743BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Treatments',
    backgroundColor: '2d89ef',
    textColor: 'FFFFFF',
    services: [
      { id: '843BDC9051', title: 'Colore Future', qualified: true },
      { id: '8630D59354', title: 'colore moda estate', qualified: false },
      { id: '863ED59357', title: 'Colore senza amm.', qualified: false },
      { id: '823FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '8D3FD490', title: 'Deco', qualified: false },
      { id: '8738DA9559', title: 'prova', qualified: false },
      { id: '843BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Man',
    backgroundColor: 'f88b17',
    textColor: 'FFFFFF',
    services: [
      { id: '943BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '9638DB9B', title: 'Colore completo', qualified: true },
      { id: '943BDC9051', title: 'Colore Future', qualified: true },
      { id: '9738DA9559', title: 'prova', qualified: false },
      { id: '943BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
  {
    title: 'Various',
    backgroundColor: 'd6006b',
    textColor: 'FFFFFF',
    services: [
      { id: '1043BDC9056', title: 'Colore 10 minuti', qualified: true },
      { id: '10638DB9B', title: 'Colore completo', qualified: true },
      { id: '1023FDA9A', title: 'Colore senza ammonia', qualified: false },
      { id: '10D3FD490', title: 'Deco', qualified: false },
      { id: '1023AD49254', title: 'elumen', qualified: false },
      { id: '10D38DE9559', title: 'Prova', qualified: false },
      { id: '10738DA9559', title: 'prova', qualified: false },
      { id: '1043BDD9253E2', title: 'Sunshine', qualified: false },
    ],
  },
]
