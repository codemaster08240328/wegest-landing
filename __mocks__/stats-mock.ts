import { IResume } from '@/shared/@types/models/stastics'

export const resume: Record<string, IResume> = {
  month: {
    revenue: 4675.64,
    avgReceiptValue: 203.29,
    fees: 3878.77,
    serviceFees: 3806.25,
    productFees: 72.52,
    attendance: 23,
    newCustomers: 1,
    handledCustomer: 11,
    expenses: 1105.32,
    netIncome: 3570.32,
  },

  today: {
    revenue: 0,
    avgReceiptValue: 0,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 0,
    newCustomers: 0,
    handledCustomer: 0,
    expenses: 0,
    netIncome: 0,
  },

  yesterday: {
    revenue: 166,
    avgReceiptValue: 166,
    fees: 0,
    serviceFees: 0,
    productFees: 0,
    attendance: 1,
    newCustomers: 0,
    handledCustomer: 1,
    expenses: 0,
    netIncome: 166,
  },
}
