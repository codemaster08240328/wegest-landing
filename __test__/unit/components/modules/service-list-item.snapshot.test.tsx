import renderer from 'react-test-renderer'
import ServiceListItem from '@/components/modules/service-list-item'

import { categories as serviceCategories } from '@/__mocks__/service-categories-mock'

const category = serviceCategories[3]

describe('<ServiceListItem> snapshot', () => {
  it('renders properly with all required props', () => {
    expect(
      renderer
        .create(
          <ServiceListItem
            category={category}
            onClick={() => null}
            onSelectService={() => null}
          />,
        )
        .toJSON(),
    ).toMatchSnapshot()
  })

  it('renders with a class', () => {
    expect(
      renderer
        .create(
          <ServiceListItem
            className='class-test'
            category={category}
            onClick={() => null}
            onSelectService={() => null}
          />,
        )
        .toJSON(),
    ).toMatchSnapshot()
  })
})
