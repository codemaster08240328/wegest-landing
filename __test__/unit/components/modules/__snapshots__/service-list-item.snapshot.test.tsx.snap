// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServiceListItem> snapshot renders properly with all required props 1`] = `
<div
  className="MuiPaper-root MuiAccordion-root WithStyles(ForwardRef(Accordion))-root-3 makeStyles-root-1  MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    aria-controls="Meches-content"
    aria-disabled={false}
    aria-expanded={false}
    className="MuiButtonBase-root MuiAccordionSummary-root WithStyles(ForwardRef(AccordionSummary))-root-5"
    id="Meches-header"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    style={
      Object {
        "backgroundColor": "#ffb300",
        "color": "#FFFFFF",
      }
    }
    tabIndex={0}
  >
    <div
      className="MuiAccordionSummary-content WithStyles(ForwardRef(AccordionSummary))-content-6"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
        data-testid="el-icon"
      >
        arrow_right
      </span>
      <h3
        className="MuiTypography-root MuiTypography-body1"
        data-testid="title"
      >
        Meches
      </h3>
    </div>
  </div>
  <div
    className="MuiCollapse-container MuiCollapse-hidden"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner"
      >
        <div
          aria-labelledby="Meches-header"
          id="Meches-content"
          role="region"
        >
          <div
            className="MuiAccordionDetails-root WithStyles(ForwardRef(AccordionDetails))-root-8"
            data-testid="services"
          >
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="443BDC9056"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore 10 minuti
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="4638DB9B"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore completo
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="443BDC9051"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore Future
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="4630D59354"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  colore moda estate
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="463ED59357"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore senza amm.
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="423FDA9A"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore senza ammonia
                </span>
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<ServiceListItem> snapshot renders with a class 1`] = `
<div
  className="MuiPaper-root MuiAccordion-root WithStyles(ForwardRef(Accordion))-root-3 makeStyles-root-1 class-test MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
>
  <div
    aria-controls="Meches-content"
    aria-disabled={false}
    aria-expanded={false}
    className="MuiButtonBase-root MuiAccordionSummary-root WithStyles(ForwardRef(AccordionSummary))-root-5"
    id="Meches-header"
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    role="button"
    style={
      Object {
        "backgroundColor": "#ffb300",
        "color": "#FFFFFF",
      }
    }
    tabIndex={0}
  >
    <div
      className="MuiAccordionSummary-content WithStyles(ForwardRef(AccordionSummary))-content-6"
    >
      <span
        aria-hidden={true}
        className="material-icons MuiIcon-root"
        data-testid="el-icon"
      >
        arrow_right
      </span>
      <h3
        className="MuiTypography-root MuiTypography-body1"
        data-testid="title"
      >
        Meches
      </h3>
    </div>
  </div>
  <div
    className="MuiCollapse-container MuiCollapse-hidden"
    style={
      Object {
        "minHeight": "0px",
      }
    }
  >
    <div
      className="MuiCollapse-wrapper"
    >
      <div
        className="MuiCollapse-wrapperInner"
      >
        <div
          aria-labelledby="Meches-header"
          id="Meches-content"
          role="region"
        >
          <div
            className="MuiAccordionDetails-root WithStyles(ForwardRef(AccordionDetails))-root-8"
            data-testid="services"
          >
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="443BDC9056"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore 10 minuti
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="4638DB9B"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore completo
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="443BDC9051"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore Future
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="4630D59354"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  colore moda estate
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="463ED59357"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore senza amm.
                </span>
              </label>
            </div>
            <div
              className="makeStyles-root-9 makeStyles-checkbox-2"
              data-testid="checkboxContainer"
            >
              <label
                className="MuiFormControlLabel-root"
              >
                <span
                  aria-disabled={false}
                  className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-11 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                  data-testid="checkbox"
                  onBlur={[Function]}
                  onDragLeave={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex={null}
                >
                  <span
                    className="MuiIconButton-label"
                  >
                    <input
                      checked={false}
                      className="PrivateSwitchBase-input-14"
                      data-indeterminate={false}
                      name="423FDA9A"
                      onChange={[Function]}
                      type="checkbox"
                    />
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                      />
                    </svg>
                  </span>
                  <span
                    className="MuiTouchRipple-root"
                  />
                </span>
                <span
                  className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                >
                  Colore senza ammonia
                </span>
              </label>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
